content/doc/administrate/scalability.md

```diff
@@ -51,8 +51,6 @@ The following scheme depicts a Scaler replication in case of a load increase:
 
 {{< image "/images/doc/scaling_horizontal_scheme.jpg" "You can manage the range of Scalers you consider in the application configurator. The range is from 1 to 40." >}}
```

Comment:
While checking this with the review app, I think that "the range is 1 to 40" is too small as a caption. I'm not sure where to put it, but currently, it's the only place that has this information. I think we should display this information differently.

For starters:

```suggestion
{{< image "/images/doc/scaling_horizontal_scheme.jpg" "Horizontal scaling: you can define the min and max numbers of Scalers you need." >}}
```

Reply:
I'll put the 1 to 40 elsewhere, in another commit.